src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataIT.java

```diff
@@ -1133,7 +1133,7 @@ public void testGetProcedureColumns() throws Exception {
       assertFalse(resultSet.next());
       /* Call getProcedureColumns with specified procedure name and column name. This will return 2 rows: 1 for the
       result values of the procedure, and 1 for the parameter values for the specified parameter. */
-      resultSet = metaData.getProcedureColumns(null, null, "stproc1", "param2");
+      resultSet = metaData.getProcedureColumns(null, null, "STPROC1", "PARAM2");
```

Comment:
This needed to be in caps. Previously it was falsely returning results because of the lack of schema filtering.